packages/fxa-content-server/app/scripts/templates/pair/auth_complete.mustache

```diff
@@ -9,11 +9,17 @@
     <div class="graphic {{ graphicId }}" role="img" aria-label="{{#t}}Successfully connected{{/t}}"></div>
 
     <h2>{{#t}}You are now syncing with:{{/t}}</h2>
-
-    {{{ unsafeDeviceBeingPairedHTML }}}
-
+    <h2 class="family-os">{{#unsafeTranslate}}%(deviceFamily)s on %(deviceOS)s{{/unsafeTranslate}}</h2>
```

Comment:
Quick fly-by comment, should this maybe be a `span` within the h2?

(Thanks for getting a PR up for this to get us a head start next sprint!)